src/main/java/games/strategy/engine/data/PlayerID.java

```diff
@@ -13,6 +13,8 @@
   private static final long serialVersionUID = -2284878450555315947L;
   private final boolean m_optional;
   private final boolean m_canBeDisabled;
+  private final boolean isDefaultAi;
+  private final boolean isNonPlayable;
```

Comment:
Just want to confirm that this will not break save game compatibility...  As both of these fields will not exist in an old save game, they will not be set when deserializing a `PlayerID`, and they will retain their default `false` value.  Thus, they will match the default values used in the two-arg constructor, and everything should just work, right?